mocca-resiliency4j/src/main/java/com/paypal/mocca/client/MoccaResilience4j.java

```diff
@@ -126,5 +129,43 @@ public Builder bulkhead(final Bulkhead bulkhead) {
             this.feignBuilder = feignBuilder.withBulkhead(bulkhead);
             return this;
         }
+
+        /**
+         *
+         * @param fallBack the fallback object to be set to in this builder
```

Comment:
Please, add proper javadoc. Everything must be set, method description, and all tags, including `return`.

Do the same for all public methods you added here.

Reply:
Added javadoc for all newly added APIs
Also updated the end user documentation

Reply:
Thank you